Tests/Tests/STPRedirectContextTest.m

```diff
@@ -13,18 +13,42 @@
 #import "STPFixtures.h"
 #import "STPRedirectContext.h"
 #import "STPURLCallbackHandler.h"
+#import "STPWeakStrongMacros.h"
 
 @interface STPRedirectContext (Testing)
 - (void)unsubscribeFromNotifications;
 - (void)dismissPresentedViewController;
 @end
 
 @interface STPRedirectContextTest : XCTestCase
-
+@property (nonatomic, weak) STPRedirectContext *weak_sut;
 @end
 
+/*
+ NOTE:
+
+ If you are adding a test unsubscribe from notifications is called before your
```

Comment:
maybe "If you are adding a test use `unsubscribeContext:` to unsubscribe the context being tested before the test ends"

Reply:
Ah yeah I also accidentally a word here. Should be "make sure unsubscribe from notifications is called". You theoretically don't need to manually call it if your test is already verifying it is called (Which many do). You can also just directly call unsubscribe on it if you aren't explicitly OCMRejecting that method instead of using `unsubscribeContext:`. I'll rewrite this comment to be more clear.